pkg/scheduler/metrics/job.go

```diff
@@ -0,0 +1,34 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto" // auto-registry collectors in default registry
+)
+
+var (
+	jobShare = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: VolcanoNamespace,
+			Name:      "job_share",
+			Help:      "Share for one job",
```

Comment:
add more description

Reply:
How about "DRF share for one job"?

Reply:
IMO, that should be enough for the first version :)

We should not highlight plugin in description, as it dependent on the configuration.